### DISCUSSION TITLE
Does not request a password for encrypted storage

### DISCUSSION THREAD

**Discussion by zerotwoo002** [iterate-ch/cyberduck#15014] [Q&A]
I create a vault with a password, but when I open it, the password is not requested and all files are hidden

**Comment by dkocher** [comment]
Please refer to the documentation ^1] how to unlock a vault.

[^1]: https://docs.cyberduck.io/cryptomator/#unlock-vault

**Reply by zerotwoo002** [reply]
I did everything according to the documentation, but I am never asked for a password

**Comment by zerotwoo002** [comment]
Can anyone help?
